pages/form-builder/responses/[[...params]].tsx

```diff
@@ -333,13 +336,9 @@ export const getServerSideProps: GetServerSideProps = async ({
       FormbuilderParams.initialForm = initialForm;
       vaultSubmissions.push(...allSubmissions.submissions);
 
-      const isNagwareEnabled = await checkOne("nagware");
```

Comment:
If we're remove the flag check we should also remove the "nagware" flag from initialization.

Reply:
Sorry, yes I meant we should remove the flag completely from the app.

Reply:
Removed.
https://github.com/cds-snc/platform-forms-client/pull/2281/commits/4a97f1c9f6f7c37df0e19999d3aa60998dc77fe6